src/main/kotlin/com/amazon/opendistroforelasticsearch/indexmanagement/indexstatemanagement/transport/action/removepolicy/RemovePolicyAction.kt

```diff
@@ -0,0 +1,10 @@
+package com.amazon.opendistroforelasticsearch.indexmanagement.indexstatemanagement.transport.action.removepolicy
+
+import org.elasticsearch.action.ActionType
+
+class RemovePolicyAction private constructor() : ActionType<RemovePolicyResponse>(NAME, ::RemovePolicyResponse) {
+    companion object {
+        val INSTANCE = RemovePolicyAction()
+        val NAME = "cluster:admin/ism/managedindex/remove"
```

Comment:
Should we prefix it with opendistro too?

Reply:
yes, better with `opendistro-ism`